manifests/java.yml

```diff
@@ -920,7 +920,7 @@ tests/:
     test_dynamic_configuration.py:
       TestDynamicConfigHeaderTags: missing_feature
       TestDynamicConfigTracingEnabled: missing_feature
-      TestDynamicConfigV1: v1.17.0
```

Comment:
I think it's worth splitting the test out into a separate class so we can differentiate when each was added. Maybe a `TestDynamicConfigIgnoreMismatchingTargets`?